doc/source/notebooks/gpflow2_upgrade_guide.md

```diff
@@ -127,17 +151,23 @@ The `defer_build` context manager has been removed. References to it can simply
 GPflow methods that used the `@autoflow` decorator, like for example `predict_f` and `predict_y`, will previously have returned NumPy Arrays. These now return TensorFlow tensors. In many cases these can be used like NumPy arrays (they can be passed directly to many of NumPy’s functions and even be plotted by matplotlib), but to actually turn them into numpy arrays you will need to call `.numpy()` on them.
 
 For example:
-
-![predict functions return tf.Tensors, need to call .numpy() to obtain NumPy array](numpy_new.png)
+```diff
+ def _predict(self, features: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
+     mean, variance = self._model.predict_f(features)
+-    return mean, variance
++    return mean.numpy(), variance.numpy()
+```
 
 
 ## Parameter Values
 
 GPflow’s `Parameter.value` has changed from a property to a method.
 
 For example:
-
-![Parameters now behave like tensors](param.png)
+```diff
+-std_dev = np.sqrt(model.likelihood.variance.value)
++std_dev = np.sqrt(model.likelihood.variance.value())
```

Comment:
What is the difference between
```
model.likelihood.variance.value()
```
and 
```
model.likelihood.variance.numpy()
```
?

Reply:
Oh, `.value()` returns a tensor (basically a constant tensor that's equivalent to the value of the parameter), whereas `.numpy()` converts it to a numpy ndarray. Sorry, I hadn't seen this comment :( will make a quickfix.